### PR TITLE
ADEN-11157 Moving TLB under global nav and implementing UAP 4.0

### PR DESCRIPTION
**Pull request by dmnsobczak**
Ticket: https://wikia-inc.atlassian.net/browse/ADEN-11157

### CODE REVIEW

**Review by Tomasz-at-Fandom — APPROVED** [review]

File: platforms/ucp-mobile/styles.scss [hunk 1]
```diff
@@ -35,6 +35,32 @@
 		}
 	}
 
+	.uap-light {
+		$global-navigation-width: 55px;
```
Comment (preludek) [inline platforms/ucp-mobile/styles.scss:39]: this variable already exists in _uap-variables.scss

File: platforms/ucp-mobile/templates/bfaa-3-template.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ import { Observable } from 'rxjs';
 import { registerUcpMobileUapDomElements } from './configs/register-ucp-mobile-uap-dom-elements';
 import { BfaaUcpMobileConfigHandler } from './handlers/bfaa/bfaa-ucp-mobile-config-handler';
 
-export function registerBfaaNewTemplate(registry: TemplateRegistry): Observable<TemplateAction> {
+export function registerBfaa3Template(registry: TemplateRegistry): Observable<TemplateAction> {
```
Comment (preludek) [inline platforms/ucp-mobile/templates/bfaa-3-template.ts:40]: Took me a second to understand which UAP is which. I hope tests go smoothly and we can that up 🙈

**Review by preludek — APPROVED** [review]

File: platforms/ucp-mobile/styles.scss (hunk 1 above)
Comment (Tomasz-at-Fandom) [reply platforms/ucp-mobile/styles.scss]: Cool, I used the existing variable :)

File: platforms/ucp-mobile/templates/bfaa-3-template.ts (hunk 2 above)
Comment (Tomasz-at-Fandom) [reply platforms/ucp-mobile/templates/bfaa-3-template.ts:40]: Agreed :)

**Pull request merged**